scripts/rpi-jlink-server/README.md

```diff
@@ -0,0 +1,33 @@
+# Raspberry Pi J-Link Server
+
+This directory contains two scripts that are used to run tests on a remote Raspberry Pi, which is connected to a J-Link debugger:
+
+- The `request_resource.sh` script, which should be used inside a GitHub Action Workflow before and after running tests on the device.
+- The `twister_pty.py` script, which is used with the `west/east twister` command. This script receives and prints the outputs generated by the device.
+
+## Remote Raspberry Pi
+
+Documentation on how to set up the Raspberry Pi and a detailed explanation of the functionality can be found [here](https://github.com/IRNAS/irnas-runners-software/blob/main/rpi-jlink-server/README.md).
+
+## GitHub Actions Workflow
+
+The scripts mentioned are used by the `twister-rpi.yaml` GitHub Actions Workflow.
+The workflow is not included in this repository, as it is opt-in. It can be found [here](...). To include it, you must copy the workflow file to your repository and add the following GitHub secrets:
```

Comment:
Be more specific, some like this would help the future reader:
```
To include it, you must copy the `twister-rpi.yaml` from the linked repository into `.github/workflows` folder inside your repository and add the following GitHub secrets:
```